compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -67,6 +67,7 @@ class ScalaSettings extends Settings.SettingGroup {
 
   /** Scala.js-related settings */
   val scalajsGenStaticForwardersForNonTopLevelObjects: Setting[Boolean] = BooleanSetting("-scalajs-genStaticForwardersForNonTopLevelObjects", "Generate static forwarders even for non-top-level objects (Scala.js only)")
+  val scalajsMapSourceURI: Setting[List[String]] = MultiStringSetting("-scalajs-mapSourceURI", "uri1[->uri2]", "rebases source URIs from uri1 to uri2 (or to a relative URI) for source maps (Scala.js only)")
```

Comment:
To simplify cross-compilation and not have to update the existing documentation, would it make sense to make `-P:scala:foo` an alias of `-scalajs-foo` ?

Reply:
How would we do that? `-P` is a generic system to pass options to a plugin. We would have to special case `scalajs` in several places.

Reply:
I haven't looked at how -P works so I can't say, but it seems like even if it's a bit hacky it would be worth it, but I don't know, your choice!